### PR TITLE
feat: implement score sharing to twitter

### PR DESCRIPTION
**Pull request by lesnitsky**

### CODE REVIEW

File: src/game/containers/SocialSharingContainer.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import { createStructuredSelector } from 'reselect';
+
+import { hasScore, totalTime, playerPosition, playersCount } from '../selectors/player-score-selectors';
+
+import { puzzlesCount } from '../../shared/selectors/puzzle-selectors';
+import { AuthButton } from '../../components/AuthButton/AuthButton';
+
+function buildTweetText(totalTime, playerPosition, playersCount, puzzlesCount) {
+    let message = `Today I've spent ${totalTime} seconds to solve ${puzzlesCount} puzzles in #CSSQD by @2123io`;
+    if (playerPosition <= 10) {
```
Comment (kirilknysh) [inline src/game/containers/SocialSharingContainer.js]: And... `else`?

Comment (lesnitsky) [reply src/game/containers/SocialSharingContainer.js]: I'd rather not tweet anything related to my position in score if it's not so good

Comment (kirilknysh) [reply src/game/containers/SocialSharingContainer.js]: But we still display the button, right? Once user presses it - we'll tweet `undefined`, I believe

Comment (lesnitsky) [reply src/game/containers/SocialSharingContainer.js]: fixed

**Pull request merged**